docs/howto/openstack/nova/new-vgpu-server.md

```diff
@@ -0,0 +1,170 @@
+# Creating new servers with virtual GPU support
+
+In {{brand}}, you can instantiate virtual machines --- or simply
+_servers_ --- with virtual GPU (vGPU) support. For that, you use the
+`openstack` CLI utility.
+
+## Prerequisites
+
+Although not strictly necessary, it is certainly helpful to have some
+familiarity with [creating servers in {{brand}}](new-server.md),
+especially with the OpenStack CLI.
+
+## Instantiating a vGPU server
+
+Before you begin, please make sure to source [your `openstack` RC file
+first](../../getting-started/enable-openstack-cli.md).
+
+As we discussed in the [guide on creating servers](new-server.md), an
+`openstack` command for instantiating a new server looks like this:
+
+```console
+openstack server create \
+    --flavor $FLAVOR_NAME \
+    --image $IMAGE_NAME \
+    --boot-from-volume $VOL_SIZE \
+    --network $NETWORK_NAME \
+    --security-group $SEC_GROUP_NAME \
+    --key-name $KEY_NAME \
+    --wait \
+    $SERVER_NAME
+```
+
+For servers with vGPU support, the only acceptable value for variable
+`IMAGE_NAME` is `Ubuntu 22.04 NVGRID x86_64`. Additionally, the value
+for variable `FLAVOR_NAME` should be a flavor name [prefixed by
+`g`](../../../reference/flavors/index.md#compute-tiers). To see all such
+flavors, you can type something like the following:
+
+```console
+$ openstack flavor list -c Name | grep "g\."
+
+| g.8c48gb50                 |
+| g.24c120gb50               |
+| g.48c240gb                 |
+| g.12c48gb50                |
+| g.16c64gb50                |
+| g.12c64gb50                |
+| g.16c84gb50                |
+| g.8c32gb50                 |
+```
```

Comment:
If you add `-f value` to your `openstack` invocation, you can rid your screen dump of the ugly `|` characters. :)